tests/apiv2/observables.py

```diff
@@ -13,13 +12,29 @@ class ObservableTest(unittest.TestCase):
     def setUp(self) -> None:
         database_arango.db.clear()
 
+    def assert_empty_file_observable(self, data):
+        self.assertEqual(data["name"], "empty")
+        self.assertEqual(data["type"], "file")
+        self.assertEqual(data["size"], 0)
+        self.assertEqual(data["sha256"], "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855")
+        self.assertEqual(data["sha1"], "da39a3ee5e6b4b0d3255bfef95601890afd80709")
+        self.assertEqual(data["md5"], "d41d8cd98f00b204e9800998ecf8427e")
+        self.assertEqual(data["mime_type"], "inode/x-empty; charset=binary")
```

Comment:
I'd rather have duplicate code in tests rather than add logic to tests.

Reply:
Fixed in 21a7b4b4cbcaab862c75a43ef9cc82cc39c4d5dd